### PR TITLE
[16.0][MIG] account_invoice_report_payment_info: Migration to version 16.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT42418

@CarlosRoca13 please review!

ping @sergio-teruel

### CODE REVIEW

File: account_invoice_report_payment_info/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.move"
+
+    def _get_reconciled_info_JSON_values(self):
+        res = super()._get_reconciled_info_JSON_values()
```
Comment (CarlosRoca13) [inline account_invoice_report_payment_info/models/account_invoice.py]: This method does not exist anymore. So this is no having effect. 

Check where is the data collected on the report to replicate the behaviour.

**Review by CarlosRoca13 — CHANGES_REQUESTED** [review]

**Review by CarlosRoca13 — COMMENTED** [review]
It works, but I'm not sure if the code is achieving the same result as in the latest versions.

Can you confirm @carlosdauden

**Review by sergio-teruel — APPROVED** [review]
Functional and code review ok

**Comment by sergio-teruel** [comment]
ping @juangomezzurita

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**